index.html

```diff
@@ -360,6 +360,10 @@ <h2>The <dfn>ThingDescription</dfn> type</h2>
             are missing from |td|, [= exception/throw =] a
             {{"TypeError"}} and abort these steps.
           </li>
+          <li>
+            Fill any missing mandatory property defined in [[!WOT-TD]] that have a 
+            <a href="https://www.w3.org/TR/2019/CR-wot-thing-description-20190516/#sec-default-values">default definition</a> with its default.
```

Comment:
Suggestion: usually these complex steps should be more detailed, e.g. using sub-steps with a loop, or an explicit loop, for instance

> For all mandatory properties defined in [[!WOT-TD]] that have default definitions, add the property to |td| if it's missing and set its value to the default value specified in [[!WOT-TD]].

Reply:
Looks like there are no mandatory properties in the TD spec that would have default values, so this text needs updating. We need an initialization policy for the default values of mandatory TD properties. I suggest we create a separate issue about that in the TD repo, then link it here and also in a new issue in this repo. Then we can resolve this and merge this PR.